hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsHttpOperation.java

```diff
@@ -151,6 +151,11 @@ public String getResponseHeader(String httpHeader) {
     return connection.getHeaderField(httpHeader);
   }
 
+  @org.apache.hadoop.thirdparty.com.google.common.annotations.VisibleForTesting
```

Comment:
Add import for org.apache.hadoop.thirdparty.com.google.common.annotations, so that this line is 
@VisibleForTesting

Reply:
done